File: Computer Science/Hardik Ahuja/Task 2/bubblesort.py
n=int(input("Enter the number of elements : "))
l=[int(input("Enter element "+str(i)+": ")) for i in range(n)]
for i in range(0,n):
	for j in range(0,n-i-1):
		if (l[j]<l[j+1]):
			l[j],l[j+1] = l[j+1],l[j]
	print("Iteration ",i,":  ",l)
	#print(l)

print(l)

File: Computer Science/Hardik Ahuja/Task 2/bubblesort.cpp
#include <iostream>
using namespace std;

void print(int *list, int n){
	for (int i=0; i<n; i++)
		cout<<list[i]<<"\t";
	cout<<endl;
}

void sort(int *list, int n){
	//int list_new = list;
	for(int i=0;i<n;i++){
		for (int j=0;j<n-i-1;j++){
			if(list[j]>list[j+1]){
				int temp = list[j];
				list[j] = list[j+1];
				list[j+1] = temp;
			}
		}
		cout<<"Iteration "<<i+1<<"\t";
		print(list,n);
	}
}

int main(){
	int n,list[10];
	cout<<"Bubble Sort"<<endl;	
	cout<<"Enter the number of elements : ";
	cin>>n;
	for(int i=0;i<n;i++){
		cout<<"Enter element "<<i<<": ";
		cin>>list[i];
	}
	sort(list,n);
	print(list,n);
}